modules/nf-core/varscan/processsomatic/meta.yml

```
@@ -0,0 +1,127 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "varscan_processsomatic"
+description: VarScan2 is a tool for variant detection in massively parallel
+  sequencing data. It can detect SNPs, indels, and copy number variations in
+  both somatic and germline samples. It is particularly useful for analyzing
+  tumor/normal sample pairs.
+keywords:
+  - variant calling
+  - germline
+  - somatic
+  - vcf
+  - variants
+  - genomics
+tools:
+  - "varscan":
+      description: "variant detection in massively parallel sequencing data"
+      homepage: "https://github.com/dkoboldt/varscan"
+      documentation: "https://dkoboldt.github.io/varscan/"
+      tool_dev_url: "https://github.com/dkoboldt/varscan"
+      doi: "10.1101/gr.129684.111"
+      licence: ["The Non-Profit Open Software License version 3.0 (NPOSL-3.0)"]
+      identifier: biotools:varscan
+
+input:
+  - - meta:
+        type: map
+        description: |
+          Groovy Map containing sample information
+          e.g. `[ id:'sample1' ]`
+    - vcf:
+        type: file
+        description: VCF file
+        pattern: "*.vcf.gz"
```

```suggestion
        pattern: "*.{snvs,indels}.vcf.gz"
```
maybe we need to specify here that they need to be named that way